mesonbuild/modules/fs.py

```diff
@@ -117,7 +117,7 @@ def is_absolute(self, state: 'ModuleState', args: T.Tuple['FileOrString'], kwarg
     def as_posix(self, state: 'ModuleState', args: T.Tuple[str], kwargs: T.Dict[str, T.Any]) -> str:
         """
         this function assumes you are passing a Windows path, even if on a Unix-like system
-        and so ALL '\' are turned to '/', even if you meant to escape a character
+        and so ALL '\\' are turned to '/', even if you meant to escape a character
```

Comment:
So the issue here is that we actually do mean `\` and that is what people reading the file see, but pydoc sees  the single-quote being escaped?

```
Help on function as_posix in module mesonbuild.modules.fs:

as_posix(self, state: "'ModuleState'", args: 'T.Tuple[str]', kwargs: 'T.Dict[str, T.Any]') -> 'str'
    this function assumes you are passing a Windows path, even if on a Unix-like system
    and so ALL '' are turned to '/', even if you meant to escape a character
```

IMO the correct solution here is to change the docstring to `f"""`